functions/imagemagick/test/index.test.js

```diff
@@ -31,40 +31,64 @@ requestRetry = requestRetry.defaults({
   retryDelay: 1000,
 });
 
-const BUCKET_NAME = process.env.FUNCTIONS_BUCKET;
-const {BLURRED_BUCKET_NAME} = process.env;
 
 const safeFileName = 'bicycle.jpg';
 const offensiveFileName = 'zombie.jpg';
-
+const BUCKET_NAME = process.env.FUNCTIONS_BUCKET;
```

Comment:
Nit: It would be nice if we just used `FUNCTIONS_BUCKET` so that we could destruct `process.env` in the same way:

- `FUNCTIONS_BUCKET`
- `BLURRED_FUNCTIONS_BUCKET`

Reply:
Agreed. From what I can see, BUCKET_NAME is used in the same because it makes sense as simply the bucket causing the function trigger. FUNCTIONS_BUCKET is defined in `.kokoro/build.sh` and has that name so the product testing config is disambiguated.